app/_src/gateway/production/running-kong/migrate-cassandra-to-postgres.md

```diff
@@ -0,0 +1,105 @@
+
+
+## Migration Approach
+The following steps should be tested in a non-production environment. Any gaps in the target state should be identified and remediated before running it in production.
+
+| Step | Name               | Description                                                                                           | Completed |
+|------|--------------------|-------------------------------------------------------------------------------------------------------|-----------|
+| 1    | Platform build     | The Green environment is built with Kong Enterprise setup using the same version as the Blue environment |           |
+| 2    | Database setup     | A new Postgres DB is built and connected to the green environment                                     |           |
+| 3    | Regression Testing | A deployment is made into the new system and regression tested. The deployment is cleaned up when testing is completed |           |
+
+
+## Preperation
+
+
+
+| Step | Name                          | Description                                                                                                 | Completed |
+|------|-------------------------------|-------------------------------------------------------------------------------------------------------------|-----------|
+| 1    | Change embargo                | A change embargo is placed on the environment preventing new deployments or configuration changes             |           |
+| 2    | Blue environment backup       | A backup is taken on the blue environment's Cassandra database and placed into redundant storage              |           |
+| 3    | Observability setup           | Dashboards are created to track the health of the new environment                                           |           |
+| 4    | Go/No go checkpoint           | Go/No go decision point for the migration execution                                                         |           |
+
+
+
+## Execution
+
+The goal of this phase is to reproduce the Kong configuration in the blue environment and perform regression/smoke testing ahead of live traffic cut-over. 
+
+
+| Step | Name                  | Description                                                                                                           | Completed |
+|------|-----------------------|-----------------------------------------------------------------------------------------------------------------------|-----------|
+| 1    | Configuration dump    | - Execute the "deck dump" command against the blue environment, for each workspace to build a YAML representation of the Kong estate<br>- Execute "deck dump --rbac-resources-only" against the blue environment for each workspace to build a YAML representation of any RBAC resources created<br>- If using the Kong Developer Portal, run the "portal fetch" command to build a representation of the Dev Portal Assets for a workspace |           |
```

Comment:
```suggestion
| 1    | Configuration dump    | - Execute the `deck dump` command against the blue environment for each workspace to build a YAML representation of the Kong estate<br>- Execute `deck dump --rbac-resources-only` against the blue environment for each workspace to build a YAML representation of any RBAC resources created<br>- If using the Kong Dev Portal, run the `portal fetch` command via the Portal CLI to build a representation of the Dev Portal assets for a workspace |           |
```
"kong estate" is a term I've never heard before. Kong configuration? Environment? Entities maybe?